app/Http/Controllers/Tools/ToolController.php

```diff
@@ -144,6 +144,7 @@ public function store(Request $request)
             'bookingLength' => 'required|integer|min:0',
             'lengthMax' => 'required|integer|min:0',
             'bookingsMax' => 'required|integer|min:1',
+            'hidden' => 'sometimes|required',
```

Comment:
```suggestion
            'hidden' => 'sometimes|boolean|required',
```

Reply:
That doesn't work because the value is only sent if checked, so it becomes impossible to make the tool hidden